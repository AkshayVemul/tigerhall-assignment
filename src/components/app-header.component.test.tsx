import { render } from '@testing-library/react'
import { AppHeader } from './app-header.component'

describe('App Header', () => {
  it('should match snapshot', () => {
    const { container } = render(<AppHeader />)

    expect(container).toMatchSnapshot()
  })
})
