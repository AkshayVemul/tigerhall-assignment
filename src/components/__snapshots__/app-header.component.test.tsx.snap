// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Header should match snapshot 1`] = `
<div>
  <div
    class="css-1whykrx"
  >
    <h2
      class="chakra-heading css-0"
    >
      Test Component
    </h2>
    <p
      class="chakra-text css-0"
    >
      Welcome
    </p>
  </div>
</div>
`;
